libs/output-mapping/src/Writer/Table/Strategy/SqlWorkspaceTableStrategy.php

```diff
@@ -53,6 +55,25 @@ public function __construct(
         $this->dataStorage = $dataStorage;
     }
 
+    /**
+     * @return MappingFromRawConfiguration[]
+     */
+    public function getMapping(array $configuration): array
+    {
+        $mapping = [];
+        foreach ($configuration['mapping'] ?? [] as $mappingItem) {
+            // Skip configurations with direct-grant unload strategy
+            // These will be handled separately and don't need processing OM
+            if (($mappingItem['unload_strategy'] ?? '') === SqlWorkspaceTableStrategy::DIRECT_GRANT_UNLOAD_STRATEGY) {
+                $this->hasDirectGrantUnloadStrategy = true;
+                continue;
+            }
+            $mapping[] = new MappingFromRawConfiguration($mappingItem);
+        }
+
+        return $mapping;
+    }
```

Comment:
Ty jo ten sideefekt tady mě přijde dost divný. IMHO by `StrategyFactory::getTableOutputStrategy` mohla dostat ty configuration, v ctoru si udělat MappingFromRawConfiguration a nastavit hasDirectGrantUnloadStrategy. Bylo by to o dost čistčí.